lxd/migrate.go

```diff
@@ -184,6 +184,7 @@ type migrationSink struct {
 	push                  bool
 	clusterMoveSourceName string
 	refresh               bool
+	convertOptions        []string
```

Comment:
In the public API you used the word "conversion" and here and below you're using the word "convert" - it would be good to remain consistent and pick one word for this functionality.